libmscore/layout.cpp

```diff
@@ -3286,6 +3286,96 @@ void layoutHarmonies(const std::vector<Segment*>& sl)
             }
       }
 
+//---------------------------------------------------------
+//   almostZero
+//---------------------------------------------------------
+
+bool inline almostZero(qreal value)
```

Comment:
Brilliant name ;-)  
A little bit of zero is sometimes almost one

Reply:
I didn't come up with this name, got it somewhere from a QT example. 
Although a little bit of zero can be almost one, in this case a maximum shift of 0.001sp is too small to be useful, so consider it as zero is fair enough here.